ddtrace/tracer/tracer.go

```diff
@@ -317,6 +354,13 @@ func (t *tracer) Extract(carrier interface{}) (ddtrace.SpanContext, error) {
 
 // flush will push any currently buffered traces to the server.
 func (t *tracer) flushPayload() {
+	if t.statsd != nil {
+		start := time.Now()
+		tags := t.statsTags()
+		t.statsd.Gauge("datadog.tracer.flush.bytes", float64(t.payload.size()), tags, 1)
+		t.statsd.Gauge("datadog.tracer.flush.traces", float64(t.payload.itemCount()), tags, 1)
+		defer func() { t.statsd.Timing("datadog.tracer.flush.duration", time.Since(start), tags, 1) }()
```

Comment:
Would it be better to get rid of `start` and just:
```
defer func(start time.Time) {
    t.statsd.Timing("datadog.tracer.flush.duration", time.Since(start), nil, 1)
}(time.Now())
```

Reply:
This change looks ok to me. I'm not sure which is clearer.